test/functional/backends/test_netssh.rb

```diff
@@ -96,21 +96,18 @@ def test_capture
       end
 
       def test_ssh_option_merge
-        verify_host_opt = if Net::SSH::Version::MAJOR >= 5
-                            { verify_host_key: :always }
-                          else
-                            { paranoid: true }
-                          end
-        a_host.ssh_options = verify_host_opt
+        keepalive_opt = { keepalive: true }
+        test_host = a_host.dup
```

Comment:
I added `.dup` here because the modifications to `a_host` were polluting other tests.